oneview/data_source_server_profile.go

```diff
@@ -742,6 +742,119 @@ func dataSourceServerProfile() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"category": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"created": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"etag": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
enclosure uri, initial scope uri  missing.

Reply:
added ```enclosure_uri``` and ```initial_scope_uris```